src/olympia/migrations/1016-prevent-duplicate-api-keys.sql

```diff
@@ -0,0 +1,22 @@
+-- First, disable all active API keys from users who have multiple active API
+-- keys. We can't know which one they expected to work, and they couldn't use
+-- them before anyway because of MultipleObjectsReturned exception.
+-- The double subquery is here to let MySQL update and select from the same
+-- table by creating a temporary one.
+UPDATE `api_key` SET `is_active` = 0 WHERE `user_id` IN (
+    SELECT `user_id` FROM (
+        SELECT `user_id` FROM `api_key`
+        GROUP BY `user_id` HAVING SUM(is_active) > 1
+        ORDER BY `user_id`)
```

Comment:
Nit: Removing the `ORDER BY` clause saves us a few ms.

Reply:
probably not even ms :)

Reply:
Actually, more than just a few:

I just tried on prod and it took ~500ms including ordering and 200ms without.